canonical-websites/cn.ubuntu.com
# Remove DB and CMS from site

Remove the admin area, the database config and all models from the app - to turn it into a stateless Django app like most of our sites.

Fixes #178. Also resolves #47 and resolves #26, as CMS will no longer be relevant.

QA
--

``` bash
./run clean
./run
```

Browse to <http://0.0.0.0:8010/internet-of-things/> and check things look just as they should.

Also play around with `./run` functions to check they haven't broken.

Note the travis test still fails, but that'll be fixed by #180
